action.yml

```diff
@@ -1,18 +1,29 @@
 name: 'Upload Pages artifact'
 description: 'A composite action that prepares your static assets to be deployed to GitHub Pages'
 inputs:
-  directory:
-    description: 'The directory in your workspace that contains the built assets'
+  path:
+    description: 'Path of the directoring containing the static assets.'
     required: true
     default: '_site/'
+  retention-days:
+    description: 'Duration after which artifact will expire in days.'
+    required: false
+    default: '1'
```

Comment:
is retention 1 day too strict? :)

Reply:
I don't think so.

Going with @chrispat's [suggestion](https://github.slack.com/archives/C02MGQ310MS/p1652275728820239?thread_ts=1652204089.877029&cid=C02MGQ310MS).

I made it an input so people can customize that if they want.

Most people really don't need to know or care that artifacts are transiting via Actions. Since we are going to start building this way private repos, we don't want to consume too much storage.